wagtail/users/templates/wagtailusers/groups/results.html

```diff
@@ -1,4 +1,4 @@
-{% extends "wagtailadmin/generic/index_results.html" %}
+{% extends "wagtailadmin/generic/listing_results.html" %}
```

Comment:
I forgot to mention this while reviewing #10524:

Not quite sure on our stance in regards to overriding the admin templates, but now that we have removed the `wagtailusers/groups/index.html` template, shall we at least mention it in the release notes? Or we could do a more comprehensive listing on the changed/removed template files later on, as per the core team discussion.

Reply:
That's a fair point - while we don't formally support overriding arbitrary templates or guarantee that they'll remain stable across releases, someone who customised `wagtailusers/groups/index.html` (and accepts the burden of maintaining that over upgrades) no longer has that option. I think it makes sense to reinstate `wagtailusers/groups/index.html` as a template that just extends `wagtailadmin/generic/index.html` with no modifications.